erpnext/buying/doctype/purchase_order_item/purchase_order_item.json

```diff
@@ -562,7 +562,7 @@
    "oldfieldname": "conversion_factor", 
    "oldfieldtype": "Currency", 
    "permlevel": 0, 
-   "precision": "", 
+   "precision": "9", 
```

Comment:
a configurable in `Stock Settings` (that can default to the value of `9`)? Set in buying and selling controllers?

Reply:
Hmm. I would prefer if this is the default. Users can always configure field precision themselves anyway. The reason why I would rather keep `9` as default is that this should not have any breaking change but only prevent a precision loss as I explained in the description above.

Reply:
Another reason that I decided to use precision: 9 is that UOM Conversion Factor DocType also uses it